gulpfile.js

```diff
@@ -140,4 +146,26 @@ gulp.task('image_min', function() {
     .pipe(plumber())
     .pipe(image_min(options))
     .pipe(gulp.dest('./'))
-});
\ No newline at end of file
+}); 
+
+var imgInput = './content/images/**/*.{jpg,png}';
+var imgOutput = './content/images/';
+
+gulp.task('image_min', function() {
+  return gulp.src(imgInput)
+    .pipe(plumber())
+    .pipe(size())
+    .pipe(imagemin())
+    .pipe(rename({
+      suffix: "_preview"
+    }))
+    .pipe(img_resize({
+      width : 50,
```

Comment:
@nathakits - why not make the preview image super small like 20 x 20 - or even smaller like 10 x 10. The image will be blurred anyways. What is the lower bound for the preview image so that the blurred preview still looks "good" or makes a smooth transition to the high quality image.